src/Core/src/Platform/KeyboardExtensions.cs

```diff
@@ -0,0 +1,156 @@
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.Maui.Dispatching;
+using System.Threading.Tasks;
+using System.Threading;
+using System;
+
+#if IOS || MACCATALYST
+using PlatformView = UIKit.UIView;
+#elif ANDROID
+using PlatformView = Android.Views.View;
+#elif WINDOWS
+using PlatformView = Microsoft.UI.Xaml.FrameworkElement;
+#elif TIZEN
+using PlatformView = Tizen.NUI.BaseComponents.View;
+#elif (NETSTANDARD || !PLATFORM) || (NET6_0_OR_GREATER && !IOS && !ANDROID && !TIZEN)
+using PlatformView = System.Object;
+using IPlatformViewHandler = Microsoft.Maui.IViewHandler;
+#endif
+
+namespace Microsoft.Maui.Platform;
```

Comment:
Hmmm I'm wondering if we want the extensions against `ITextInput` to be inside the `Microsoft.Maui` namespace so they are just by default exposed opposed to needing to pull in `Microsoft.Maui.Platform`. I realize that create of a naming conflict. 

What if for the public facing one we just name it `SoftInputExtentions`